### ISSUE TITLE
routing - ignore the next x kilometers

### ISSUE THREAD

**Issue by osmvermesser** [osmandapp/Osmand#5208]
Basic situation: you are Routing and front of you is a temp. roadblock. when you manually leave the way the routing will bring you back to this way.

it would be nice to have a function to block the current routing for the next x (buttons for = 1,5,10 km) Kilometer. when you come back this routing block will no more active.

Not to be confused with the function where routes are permanently excluded from routing. (existing osmand-function)

regards Jan

**Comment by bitboy85** [comment]
Agree. This function should be accessible with few (2) clicks and also have a default of 5 km because often is there is no Information how long the block really is.

**Comment by osmvermesser** [comment]
a button optional send to desktop. 

area is available when use a tablet.

best define the distanze 1 or 2 possible (buttons) in properties.

regards Jan

**Comment by vshcherb** [comment]
As of today, we advise to use "Avoid road" functionality it will speed up to catch navigation engine a correct route.
